src/main/kotlin/de/twintorx/battleship/ui/Player.kt

```diff
@@ -192,14 +206,37 @@ class Player {
         updateGameBoard(client.waitForIncomingShot())
     }
 
-    private fun printBoards() {
+    private fun printBoards(inGame: Boolean = true) {
+        Writer.println("\n${PlayerMessage.AVAILABLE_SHIPS}")
+        var line = ""
+        Ship.values().forEach { line += "${it.color.paint(it.type)}(Size:${it.size})   " }
+        Writer.println(line)
```

Comment:
Writer.println("   ".join(Ship.values().forEach {
    "${it.color.paint(it.type)}(Size:${it.size})"
}))